frontends/concrete-python/tests/execution/test_matmul.py

```diff
@@ -335,6 +336,12 @@ def test_zero_matmul(bit_width, signed, helpers):
     Test matmul where one of the operators is all zeros.
     """
 
+    # FIXME: it fails
```

Comment:
this one is a strange problem @BourgerieQuentin @youben11 @aPere3 
would you know why it fails sometimes on macOS?

Reply:
so it's the only real bug I am not able to fix

Reply:
I'll have a look.

Reply:
https://github.com/zama-ai/concrete-internal/issues/879